### ISSUE TITLE
[5.x]: Namespacing does not affect aria-owns attribute

### ISSUE THREAD

**Issue by mmikkel** [craftcms/cms#18577]
### Description

Namespacing in Twig does not work with `aria-owns` attributes.

Example:

```twig
{% namespace('foobar') %}
    <div aria-owns="menu"></div>
    <div id="menu"></div>
{% endnamespace %}
```

The above template will render as

```html
<div aria-owns="menu"></div>
<div id="foobar-menu"></div>
```

...while the expected output would be

```html
<div aria-owns="foobar-menu"></div>
<div id="foobar-menu"></div>
```

Namespacing does work with other `aria` attributes (like `aria-controls`), so hopefully support for `aria-owns` can be added.

### Steps to reproduce

1. 

### Craft CMS version

5.9.17

### Installed plugins and versions

-

**Comment by i-just** [comment]
Hi, thanks for reporting! I raised a PR for this.

**Comment by brandonkelly** [comment]
Craft 4.17.12 and 5.9.18 are out with that fix. Thanks again!
